webapp/components/webrtc/components/webrtc_notification.jsx

```diff
@@ -0,0 +1,319 @@
+                    />
+                </svg>
+            );
+
+            const rejectBtn = (
+                <svg
+                    className='webrtc-icons__cancel'
+                    xmlns='http://www.w3.org/2000/svg'
+                    width='48'
+                    height='48'
+                    viewBox='-10 -10 68 68'
+                    onClick={this.handleClose}
+                >
+                    <circle
+                        cx='24'
+                        cy='24'
+                        r='34'
+                    >
+                        <title>
+                            <FormattedMessage
+                                id='webrtc.notification.decline'
+                                defaultMessage='Decline'
+                            />
+                        </title>
+                    </circle>
+                    <path
+                        transform='scale(0.7), translate(11,10)'
+                        d='M24 18c-3.21 0-6.3.5-9.2 1.44v6.21c0 .79-.46 1.47-1.12 1.8-1.95.98-3.74 2.23-5.33 3.7-.36.35-.85.57-1.4.57-.55 0-1.05-.22-1.41-.59L.59 26.18c-.37-.37-.59-.87-.59-1.42 0-.55.22-1.05.59-1.42C6.68 17.55 14.93 14 24 14s17.32 3.55 23.41 9.34c.37.36.59.87.59 1.42 0 .55-.22 1.05-.59 1.41l-4.95 4.95c-.36.36-.86.59-1.41.59-.54 0-1.04-.22-1.4-.57-1.59-1.47-3.38-2.72-5.33-3.7-.66-.33-1.12-1.01-1.12-1.8v-6.21C30.3 18.5 27.21 18 24 18z'
+                        fill='white'
+                    />
+                </svg>
+            );
+
+            const msg = (
+                <div>
+                    <FormattedMessage
+                        id='webrtc.notification.incoming_call'
+                        defaultMessage='{username} is calling you.'
+                        values={{
+                            username
+                        }}
+                    />
+                    <div
+                        className='webrtc-buttons webrtc-icons active'
+                        style={{marginTop: '5px'}}
+                    >
+                        {answerBtn}
+                        {rejectBtn}
+                    </div>
+                </div>
+            );
+
+            return (
+                <div className='webrtc-notification'>
+                    <audio
+                        ref='ring'
+                        src={ring}
+                        autoPlay={true}
+                    />
+                    <div>
+                        {profileImg}
+                    </div>
+                    {msg}
+                </div>
+            );
+        } else if (this.state.rhsOpened && WebrtcStore.isBusy()) {
+            return (
+                <div
+                    className='webrtc-notification__rhs'
```

Comment:
@asaadmahmood check this out, I created this class so you can style the ongoing call indicator